### PR TITLE
epic: kmcompx - kmcmpdll cross-platform compiler 🚑

### PR DESCRIPTION
**Pull request by mcdurdin**
@keymanapp-test-bot skip

### CODE REVIEW

**Review by SabineSIL — APPROVED** [review]

**Comment by mcdurdin** [comment]
Blocked by #8541; will merge if all tests pass once that merges.

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.84-alpha](https://keyman.com/downloads/releases/alpha/17.0.84)

**Pull request merged**
